README.md

```diff
@@ -89,30 +97,33 @@ $multipleValidations = new MultipleValidationWithAnd([
 $validator->isValid("example@ietf.org", $multipleValidations); //true
 ```
 
-#### Additional validations ####
-Validations not present in the RFCs 
+#### Additional validations
 
-1. [SpoofCheckValidation](/src/Validation/Extra/SpoofCheckValidation.php): Will check for multi-utf-8 chars that can signal an erroneous email name.
+Validations not present in the RFCs
 
+1. [SpoofCheckValidation](/src/Validation/Extra/SpoofCheckValidation.php): Will check for multi-utf-8 chars that can
+   signal an erroneous email name.
 
-### How to extend ###
+### How to extend
 
-It's easy! You just need to implement [EmailValidation](/src/Validation/EmailValidation.php) and you can use your own validation.
+It's easy! You just need to implement [EmailValidation](/src/Validation/EmailValidation.php) and you can use your own
+validation.
 
-## Contributing ##
+## Contributing
 
 Please follow the [Contribution guide](CONTRIBUTING.md). Is short and simple and will help a lot.
 
-## Other Contributors ##
+## Other Contributors
 
 (You can find current contributors [here](https://github.com/egulias/EmailValidator/graphs/contributors))
 
 As this is a port from another library and work, here are other people related to the previous one:
 
-* Ricard Clau [@ricardclau](https://github.com/ricardclau):      	Performance against PHP built-in filter_var (v2 and earlier)
```

Comment:
Please keep this list in it's original format.

Reply:
Refer to https://github.com/egulias/EmailValidator/pull/364#discussion_r1212802161.